File: frontend/src/components/model/RoomModel.ts
import {Category} from "./Category.ts";

export type RoomModel = {
    id: string;
    name: string;
    address: string;
    category: Category;
    description: string;
    appUserGithubId: string;
    appUserUsername: string;
    appUserAvatarUrl: string;
    appUserGithubUrl: string;
    isActive: boolean;
    imageUrl: string;
}

File: backend/src/main/java/ropold/backend/model/RoomModel.java
package ropold.backend.model;

public record RoomModel(
        String id,
        String name,
        String address,
        Category category,
        String description,
        String appUserGithubId,
        String appUserUsername,
        String appUserAvatarUrl,
        String appUserGithubUrl,
        boolean isActive,
        String imageUrl
) {
}
